raiden_contracts/tests/test_channel_settle_unlock_state.py

```diff
@@ -258,13 +258,16 @@ def f(
 
 
 @pytest.mark.parametrize('channel_test_values', channel_settle_test_values)
-def test_channel_settle_valid_old_balance_proof_values(
+@pytest.mark.parametrize('tested_range', ('valid', 'old1', 'old2'))
```

Comment:
From the context of the test, `valid`, `old1` and `old2` are not clear enough. I suggest something like `one_old`, `both_old_1`, `both_old2`.

Reply:
Done.